.Rhistory

```diff
@@ -0,0 +1,235 @@
+library(reticulate)
```

Comment:
The history of your R session should not be part of the package. The history presents the list of the command lines you've run. If you want to save them, they need to go in a cleaned R script.